src/mixins/validation.js

```diff
@@ -50,9 +50,10 @@ export const validation = {
         if (Array.isArray(form[key])) {
           for (let i = 0; i < form[key].length; i += 1) {
             for await (const subKey of Object.keys(form[key][i])) {
-              valid = await this.$refs[`${key}${i}_${subKey}`][0].validate() && valid
+              console.log(await this.$refs[`${key}.${i}.${subKey}`])
```

Comment:
Hi @arsenijesavic, could you remove please all console logs, the gitflow will fail if we merge a console

Reply:
Yes!